docs/stratis.txt

```diff
@@ -49,6 +49,10 @@ pool add-cache <pool_name> <blockdev> [<blockdev>..]::
 	 Add one or more blockdevs to an existing pool, to be used as a cache
 	 instead of additional storage. Typically, smaller and faster drives,
 	 such as SSDs, are used for this purpose.
+         NOTE: There is a temporary restriction on the total size of the
+         cache device of 32 TiB. Adding devices to the cache so that the
+         cumulative size of all the devices in the cache exceeds 32 TiB will
+         result in the engine returning an error.
```

Comment:
I don't think command line users should care about knowing that there is an `engine` and that it's returning as error.  IMHO simply stating `... will result in an error.` would be fine.

Reply:
Sure.